### DISCUSSION TITLE
Neutering the Narrator

### DISCUSSION THREAD

**Discussion by Primitive-Human** [ACGaming/UniversalTweaks#50] [Bugfix Suggestions]
As Mojang is so up its own arse about not wanting to disable the keybind for the Narrator system, I'll have to resort to other methods to do that.

We got 2 options: 

1. [Rebind Narrator](https://www.curseforge.com/minecraft/mc-mods/rebind-narrator)  (1.12.2), even though I'm not well-versed in what its license allows;
2. Prevent the Narrator library (I believe text2speech) from loading alltogether, _somehow_. 

Sure, it being an accessability feature is nice, unless you look at the vast majority of players who don't even need a robot narrator.

**Comment by ACGaming** [comment]
Implemented in b9fc48ce5362ef455a46b857ad5db45b7060a6ae.
